src/utils/transactorHelper.ts

```diff
@@ -0,0 +1,51 @@
+import { Contract } from '@ethersproject/contracts'
+import { Transactor, TransactorOptions } from 'hooks/Transactor'
+import { isUndefined } from 'lodash'
```

Comment:
I want to go back and add all of the `lodash/{modules}` across the project to reduce bundle size. I can grab it later.